pkg/crc/services/dns/dns_windows.go

```diff
@@ -61,10 +60,7 @@ func setInterfaceNameserverValue(iface string, address string) {
 	exe := "netsh"
 	args := fmt.Sprintf(`interface ip set dns "%s" static %s primary`, iface, address)
 
-	err := win32.ShellExecuteAsAdmin(fmt.Sprintf("add dns server address to interface %s", iface), win32.HWND_DESKTOP, exe, args, "", 0)
-	if err != nil {
-		logging.Warnf("Error occured setting interface nameserver: %v", err)
-	}
+	_ := win32.ShellExecuteAsAdmin(fmt.Sprintf("add dns server address to interface %s", iface), win32.HWND_DESKTOP, exe, args, "", 0)
 }
```

Comment:
This reverts 2ca7c7c39647, not clear why (and this breaks compilation `:=` should be `=`)

Reply:
This belongs in a separate commit, nothing to do with the rest of the changes in that commit.

Reply:
So I do not see why a new commit changes much. The code was already under refactor here

Reply:
But this is really unrelated to the other changes. If either this or the other changes introduce a bug/regression, this is harder to bisect/revert as a single commit than when it's separate. And there is no associated logs, so hard to know by looking at git history that this was an intentional change in this commit.

Reply:
in future we will be more strict about this. we will also not in general merge before QA has signed off